src/makeInfinityAdder.js

```diff
@@ -4,7 +4,20 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(number) {
+    if (number !== undefined) {
+      sum += number;
+      return adder;
+    } else {
```

Comment:
Considering the task doesn't require checking the type of the argument, you can simplify the condition to just check for the absence of arguments. This can be achieved by using `arguments.length === 0` instead of comparing `number` with `undefined`. This makes the intent of the code clearer to readers who might not be sure what types of values are expected.